$NetBSD: patch-ac,v 1.10 2015/12/29 23:34:45 dholland Exp $

Fix handling of installation directories.
Install static libs with the right permissions.

--- Makefile.rules.orig	2013-11-14 23:51:29.000000000 +0000
+++ Makefile.rules
@@ -808,9 +808,6 @@ endif
 # in the file so they get built before dependencies
 #---------------------------------------------------------
 
-$(DESTDIR)$(PROJ_bindir) $(DESTDIR)$(PROJ_libdir) $(DESTDIR)$(PROJ_includedir) $(DESTDIR)$(PROJ_etcdir)::
-	$(Verb) $(MKDIR) $@
-
 # To create other directories, as needed, and timestamp their creation
 %/.dir:
 	$(Verb) $(MKDIR) $* > /dev/null
@@ -954,7 +951,9 @@ install-local::
 uninstall-local::
 	$(Echo) UnInstall circumvented with NO_INSTALL
 else
-install-local:: $(DESTDIR)$(PROJ_etcdir) $(CONFIG_FILES)
+PROJ_etcdir=${PREFIX}/share/examples/llvm
+install-local:: $(CONFIG_FILES)
+	$(MKDIR) ${PROJ_etcdir}
 	$(Echo) Installing Configuration Files To $(DESTDIR)$(PROJ_etcdir)
 	$(Verb)for file in $(CONFIG_FILES); do \
           if test -f $(PROJ_OBJ_DIR)/$${file} ; then \
@@ -1248,7 +1247,7 @@ install-local:: $(DestArchiveLib)
 $(DestArchiveLib): $(LibName.A) $(DESTDIR)$(PROJ_libdir)
 	$(Echo) Installing $(BuildMode) Archive Library $(DestArchiveLib)
 	$(Verb) $(MKDIR) $(DESTDIR)$(PROJ_libdir)
-	$(Verb) $(INSTALL) $(LibName.A) $(DestArchiveLib)
+	$(Verb) $(DataInstall) $(LibName.A) $(DestArchiveLib)
 
 uninstall-local::
 	$(Echo) Uninstalling $(BuildMode) Archive Library $(DestArchiveLib)
